content/source/docs/enterprise/api/run.html.md

```diff
@@ -533,6 +533,51 @@ curl \
   https://app.terraform.io/api/v2/runs/run-DQGdmrWMX8z9yWQB/actions/force-cancel
 ```
 
+## Forcefully execute a run
+
+`POST /runs/:run_id/actions/force-execute`
+
+Parameter | Description
+----------|------------
+`run_id`  | The run ID to execute
+
+The `force-execute` action is analogous to executing the latest queued run after discarding or canceling all prior runs that are not already complete. It initiates the same actions as the "Run this plan now" button at the top of the view of a pending run.
+
+This endpoint enforces the following prerequisites:
+
+* the target run is in the Pending state
```

Comment:
Should each of these sentences be upper-cased? Actually not sure but that would feel right to me.

Also in casing land, `Pending` isn't the actual name of the state, and these are API docs, so I'd suggest using the real state name of `pending`.